res/database/Default/user/cujo.xml

```diff
@@ -0,0 +1,1807 @@
+                <en>New marathon world record</en>
+            </title>
+            <description>
+                <de>Der Kenianer Danny Komitti hat einen neuen Marathon-Weltrekord aufgestellt. In Berlin lief er die 42,195 Kilometer lange Strecke in der neuen Bestzeit von 2:02:57 Stunden. Er verbesserte damit den bisherigen Weltrekord um 26 Sekunden.</de>
+                <en>Kenyan Danny Komitti has set a new marathon world record. In Berlin, he ran the 42.195 kilometer course in the new best time of 2:02:57 hours. He thus improved the previous world record by 26 seconds.</en>
+            </description>
+            <data genre="2" price="1.0" quality="80" />
+            <availability year_range_from="2014" year_range_to="2014" />
+        </news>
+        <news id="cujo-news-marathon-2018" type="0" thread_id="cujo-news-marathon-2018" creator="8827" created_by="Cujo">
+            <title>
+                <de>Neuer Marathon-Weltrekord</de>
+                <en>New marathon world record</en>
+            </title>
+            <description>
+                <de>Der Kenianer Elwood Pogchige hat einen neuen Marathon-Weltrekord aufgestellt. In Berlin lief er die 42,195 Kilometer lange Strecke in der neuen Bestzeit von 2:01:39 Stunden. Er verbesserte damit den bisherigen Weltrekord um 78 Sekunden.</de>
+                <en>Kenyan Elwood Pogchige has set a new marathon world record. In Berlin, he ran the 42.195 kilometer course in the new best time of 2:01:39 hours. He thus improved the previous world record by 78 seconds.</en>
+            </description>
+            <data genre="2" price="1.0" quality="80" />
+            <availability year_range_from="2018" year_range_to="2018" />
+        </news>
+        <news id="cujo-news-marathon-2022" type="0" thread_id="cujo-news-marathon-2022" creator="8827" created_by="Cujo">
+            <title>
+                <de>Neuer Marathon-Weltrekord</de>
+                <en>New marathon world record</en>
+            </title>
+            <description>
+                <de>Der Kenianer Elwood Pogchige hat einen neuen Marathon-Weltrekord aufgestellt. In Berlin lief er die 42,195 Kilometer lange Strecke in der neuen Bestzeit von 2:01:09 Stunden. Er verbesserte damit seinen eigenen Weltrekord um 30 Sekunden.</de>
+                <en>Kenyan Elwood Pogchige has set a new marathon world record. In the German capital Berlin, he ran the 42.195 kilometer course in the new best time of 2:01:09 hours. He thus improved his own world record by 30 seconds.</en>
+            </description>
+            <data genre="2" price="1.0" quality="80" />
+            <availability year_range_from="2022" year_range_to="2022" />
+        </news>
+		<!-- Spiele des Jahres -->
+        <news id="cujo-news-brettspiel-1985" type="0" thread_id="cujo-news-brettspiel-1985" creator="8827" created_by="Cujo">
+            <title>
+                <de>Detektivspiel ist das Brettspiel des Jahres 1985</de>

```

Comment:
Auch eine schöne Idee. Am liebsten wieder weniger realistisch aber dafür generischer für mehr Veröffentlichungen.

Mein Gefühl ist, dass in den seltensten Fällen von 1985 bis 2022 gespielt wird. Im schlimmsten Fall sieht man also eine solche Nachricht nie. Die aktuelle Jahreszahl kann als Variable eingefügt werden...

Reply:
Ich bin mir nicht sicher, ob dir klar ist, wie eine Idee für eine News entsteht. Es ist fast nie so, dass ich eine Idee habe, die für alle Jahre passt. Meistens habe ich eine Idee und stelle fest, dass sie nur für eine bestimmte Zeit funktioniert. Die Gamezworld-News würde in den 80er Jahren niemand verstehen. Soll ich die Idee dann verwerfen? Oder soll ich sie trotzdem verwenden und den Zeitpunkt der News einschränken?

Reply:
ich empfinde die "Brettspiel des Jahres"-Storyline gut und wenn man genuegend Eintraege hat, kann die ruhig auf das Jahr genau datiert werden.
Vogel des Jahres gibt es ja auch schon als Newskette.

Ich denke diese "des Jahres"-Ketten lassen sich auch aufbauen:
- Kinofilm des Jahres
- Buch des Jahres
- Sportler des Jahres
- ...

Hier waere nun eher zu fragen @nittka ob wir den Aufwand betreiben wollen solche an sich "Seriennews" gesondert in der DB definieren zu koennen. Also eine "Grundnews", dazu Variablen - und sie triggert sich immer selbst. Mit jedem "trigger" muss ein Wert ansteigen - der den Index der Variablen nutzt. Also kein "Zufall" sondern gezielter Zugriff.
Dazu muesste beim Einlesen aber sichergestellt werden, dass alle "Serienelemente" die in der Vergangenheit _laegen_ ausgeloest wurden.

Einfacher ist es da wohl, einfach die ... 20 Eintraege zu copy-paste-adjusten

Reply:
> Ich bin mir nicht sicher, ob dir klar ist, wie eine Idee für eine News entsteht.

Natürlich kann ich nicht wissen, wie genau eine Idee bei Dir entsteht, aber ich habe eine sehr klare Vorstellung davon, unter welchen Gesichtspunkten ich mir die News (auch) anschaue.

Die Nachrichten tauchen im Spiel immer wieder auf. Je mehr Wiederholungen es gibt, desto "langweiliger" (es gab schon entsprechende Meldungen im Forum). Es gibt Nachrichten, bei denen eine Wiederholung "schlimmer" ist als bei anderen (konkreter Atomreaktor explodiert, Landung auf dem Roten Platz), denn es sind Ereignisse, die eigentlich nur einmal stattgefunden haben können - im Gegensatz zur Entführung der Queen oder Streiks, bei denen auch eine Wiederholung plausibel ist. Ich habe auch nichts dagegen, dass es solche "Einmalnachrichten" gibt, aber um das Spiel interessant zu gestalten, bräuchte man von denen eben SEHR VIELE. Eine weitere hilft globalgalaktisch nicht so sehr.

Dann gibt es Nachrichten, die (in ihrer Ausgangsform) nur in einem bestimmten Zeitraum Sinn ergeben, sagen wir der Einfachheit halber 1993. Spielt jemand also mit späterem Startjahr oder startet 1985 und ist 5 Jahre später fertig, ist eine solche Nachricht "verloren". Wenn einem wichtig ist, dass in jedem Jahr auch wirklich das richtige Spiel des Jahres genannt wird, muss man für jedes Jahr eine Nachricht erstellen, die sich konsequenterweise nach dem Veröffentlichen selbst deaktiviert - denn dieses eine Spiel des Jahres wird nur einmal vergeben. Erstellt man aber eine "Vogel des Jahres"-Nachricht mit dem Jahr als Variable und 50 möglichen Vögeln, hat man mit einer Nachricht alle möglichen Spieljahre abgedeckt. Schränkt man die Veröffentlichung noch auf eine Jahreszeit oder einen Monat ein, reduziert sich die Wahrscheinlichkeit, dass sie im selben Jahr zweimal kommt. 

Ich sage nicht, dass sich die Bestandsdaten immer an diese Regel halten. Aber das heißt ja nicht, dass man die bekannten Fehler, die eigentlich mal beseitigt werden müssten, um weitere Fälle ergänzen muss.

Bei den Nachrichtenideen sind eben welche dabei, die sich bei weniger Realitätstreue für generische Nachrichten eignen würden. Um nur zwei Beispiele zu nennen

* Marathon: X läuft in Y Bestzeit/verfehlt die eigenen Erwartungen/überrascht mit persönlicher Bestleistung... schon hat man mit einigen Personen und Städten eine Nachricht, die überhaupt keine Jahreseinschränkung benötigt
* Ältester Mensch der Welt stirbt: das Alter muss im Laufe der Zeit nicht steigen, der Nächstälteste könnte auch jünger sein. Mir persönlich wäre die Konsistenz nicht so wichtig. Aber ich fände es seltsam, wenn dieselbe Person im selben Spiel mehrfach stirbt!